src/components/Textarea/Textarea.spec.tsx

```diff
@@ -9,46 +9,47 @@ const PLACEHOLDER = 'This is the placeholder';
 const DECORATOR_TEXT = 'And also a decorator';
 const DECORATOR = <span>{DECORATOR_TEXT}</span>;
 const INPUT_TEXT = 'some text';
+const ROW_HEIGHT = 16;
+const TEXTAREA_ID = 'textarea.tw-border';
```

Comment:
I think it would be better to add a data-test-id like for example `data-test-id="textarea-component"` or something like that instead of using the tag with the tailwind class. Don't you think?

Reply:
Fixed :)